src/list.js

```diff
@@ -102,9 +108,12 @@ const displayTask = () => {
     document.querySelectorAll('.btn-edit').forEach((e) => {
       e.addEventListener('click', editTask);
     });
-  } else {
-    document.querySelector('todo-list').innerHTML = '';
-  }
+    document.querySelectorAll('.check-box').forEach((e) => {
+      e.addEventListener('change', MakeComplete);
+    })
```

Comment:
Please make sure you make the `completed` styling persist even after page refresh.

<img width="794" alt="Screen Shot 2022-08-18 at 9 09 31" src="https://user-images.githubusercontent.com/82426682/185415956-38c331be-0ad3-4d64-9a4d-187f1943c477.png">

As you can see, the task is marked as `completed` but the styling doesn't reflect it. Please fix it.